Utilities/ocvalidate/Checklist.md

```diff
@@ -0,0 +1,65 @@
+ocvalidate Checklist
+=====================
+
+As of commit <TODO>, ocvalidate performs the following checks:
+
+## ACPI
+- [ ] Add->Entry[N]->Path: Only `0-9, A-Z, a-z, '_', '-', '.', '/', and '\'` are accepted.
```

Comment:
Make it `-` not `- [ ]` or change `- [ ]` to `- [ ]` for those checks that are implemented.

Reply:
For now all checks are implemented in Checklist.md, so I guess I will simply change all of them to `-`.